applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -637,6 +649,18 @@ pub async fn command_runner(
                     Err(e) => eprintln!("BurnTari error! {}", e),
                 }
             },
+            CreateKeyPair(args) => match create_key_pair(key_manager_service.clone(), args.key_seed).await {
+                Ok((sk, pk)) => {
+                    println!(
+                        "create new key combo pair: 
+                                1. secret key: {}, 
+                                2. public key {}",
+                        sk.to_hex(),
+                        pk.to_hex()
```

Comment:
The secret string is left unzeroed after it is dropped.
I think the following works:
```suggestion
                    println!(
                        "create new key combo pair: 
                                1. secret key: {}, 
                                2. public key {}",
                        Zeroizing::new(sk.to_hex()),
                        pk.to_hex()
```

Reply:
Great ! Actually `Zeroizing` does not implement the `Display` trait, add to refactor line `656` to use `{:?}`.

Reply:
Bummer, cant use debug format as that wont print out anything useful to the user.

Actually, you can do this because Zeroizing implements `Deref<T>`
`*Zeroizing::new(sk.to_hex())`